src/Schema/Schema.php

```diff
@@ -200,6 +200,11 @@ private function getFullQualifiedAssetName($name)
         return strtolower($name);
     }
 
+    private function normalizeName(AbstractAsset $asset): string
+    {
+        return $asset->getFullQualifiedName($this->getName());
```

Comment:
Why don't we switch to the recommended methods right now? That way we don't have to use `IfCalledFromOutside`, and we have an example of how to do the switch.

Reply:
Because the deprecated method may be extended and contain custom logic. Not calling is technically a BC break.